projects/vdk-core/src/vdk/internal/builtin_plugins/ingestion/ingester_router.py

```diff
@@ -221,15 +224,57 @@ def __initialize_ingester(self, method) -> IngesterBase:
                 f"If upgraded recently consider reverting to previous version. Or use another method type.",
             )
         else:
+            # Initialize the pre- and post- processors.
+            initialized_pre_processors = self.__get_initialized_processors(
```

Comment:
why not initialize processors once? the config sequences wouldn't change in runtime,
yet multiple ingesters could be initialized

Reply:
got the idea; I'm ok with that impl